### PR TITLE
feat: optimzie sidebar ads

### PR DESCRIPTION
**Pull request by ahmaxed**
⚠️ NOTE: BLOCKED UNTIL FURTHER NOTICE ⚠️ 

This pr optimizes the number of ads that can fit in the sidebar.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: src/_includes/assets/css/screen.css
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]: This makes the entire side bar gray.

Is that the intended behavior? Seems like we have a few different ways of loading GAM ads, adding in the advertisement text, and setting the background color.

Not a huge deal at all, but we might want to try consolidating the different methods in the future, or consider removing the separate background color like on the forum.

File: src/_includes/assets/css/screen.css
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]: This should get the CodeFactor test to pass:

```suggestion
    padding: 0 10px 10px;
```

File: src/_includes/assets/css/screen.css
Comment (ahmaxed) [reply src/_includes/assets/css/screen.css]: Yes, I am testing different methods to see which ones does better.

**Comment by ahmaxed** [comment]
Waiting on Quincy's approval of the banner ads before proceeding.

**Review by scissorsneedfoodtoo — DISMISSED** [review]
⚠️ NOTE: BLOCKED UNTIL FURTHER NOTICE ⚠️

Tested again locally and everything LGTM.

Approving this now so we can merge this freely after Quincy signs off on the banner ads.

**Comment by ahmaxed** [comment]
Seems like we would need a title 'advertisement' for each ad, so I made some adjustments. 
@scissorsneedfoodtoo, I would appreciate it if you could give it another go.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Tested locally and everything LGTM 👍

I like the way that the "ADVERTISEMENT" text is hidden by default now. That's a huge improvement over what we currently have. It also makes sense to show this text above each ad that loads.

**Pull request merged**
